### DISCUSSION TITLE
Exposing different ports in different services

### DISCUSSION THREAD

**Discussion by agronholm** [emqx/emqx-operator#917] [Q&A]
Is it possible to use emqx-operator to expose a TCP listener to the cluster only while exposing the SSL and WSS listeners as a load balancer service? I don't want to expose unsecured services to the outside world, but I need to be able to connect to the broker from within the cluster.

**Comment by Rory-Z** [comment]
Hi, @agronholm you can create service by yourself exposing the SSL and WSS listeners as a load balancer, just ensure the correct selection of Pod.
